containers.go

```diff
@@ -380,7 +385,7 @@ func (r *containerStore) BigData(id, key string) ([]byte, error) {
 
 func (r *containerStore) BigDataSize(id, key string) (int64, error) {
 	if key == "" {
-		return -1, errors.Wrapf(ErrInvalidBigDataName, "data name %q can not be used as a filename", key)
+		return -1, errors.Wrapf(ErrInvalidBigDataName, "data name is empty")
```

Comment:
Should we put the name of the function into the error.
Can't return BigDataSize, data name can not be empty.

Reply:
Hmm, we don't usually do that.  We're using `Wrapf` to get backtraces in error objects.